### PR TITLE
Update instructions on downloadIsisData and conda install in README

### PR DESCRIPTION
**Pull request by BerginJay**
## Description

1. Changed the `rsync` related commands in the ISIS SPICE Web Service section to `downloadIsisData` command and added some instructions.
2. Declare `channel_priority=flexible` when building an environment with conda to avoid package conflict errors

## Related Issue

#5158 
## How Has This Been Validated?

Details on how to reproduce the package conflict are in the corresponding issue #5158 
## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Documentation change (update to the documentation; no code change)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Infrastructure change (changes to things like CI or the build system that do not impact users)

## Checklist:

- [x] I have read and agree to abide by the [Code of Conduct](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/Code-Of-Conduct.md)
- [x] I have read the [**CONTRIBUTING**](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CONTRIBUTING.md) document.
- [x] My change requires a change to the documentation and I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] I have added myself to the [.zenodo.json](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/.zenodo.json) document.
- [x] I have added any user impacting changes to the [CHANGELOG.md](https://github.com/USGS-Astrogeology/ISIS3/blob/dev/CHANGELOG.md) document.

## Licensing
This project is mostly composed of free and unencumbered software released into the public domain, and we are unlikely to accept contributions that are not also released into the public domain. Somewhere near the top of each file should have these words:

> This work is free and unencumbered software released into the public domain. In jurisdictions that recognize copyright laws, the author or authors of this software dedicate any and all copyright interest in the software to the public domain.

- [x] I dedicate any and all copyright interest in this software to the public domain. I make this dedication for the benefit of the public at large and to the detriment of my heirs and successors. I intend this dedication to be an overt act of relinquishment in perpetuity of all present and future rights to this software under copyright law.

### CODE REVIEW

**Comment by BerginJay** [comment]
I noticed this before my trip on the 10th and submitted the PR without looking at it when I got back from the trip. Just saw that there is already a PR about rsync #5155 . Is it possible to merge only the single commit related to `channel_priority` or combine the rsync part of this PR with the other PR #5155 ?

File: README.md
Comment (Kelvinrr) [inline README.md:82]: We might want to add this to the Jenkinsfile so we can make sure it doesn't break anything.

**Comment by Kelvinrr** [comment]
> I noticed this before my trip on the 10th and submitted the PR without looking at it when I got back from the trip. Just saw that there is already a PR about rsync #5155 . Is it possible to merge only the single commit related to `channel_priority` or combine the rsync part of this PR with the other PR #5155 ?

You would need to update your PR to only include those changes. The other PR is waiting for merge conflict changes. I think if we get channel priority change in the Jenkinsfile and tests come black clean we can approve and merge this. We can favor whoever gets the PR merged first and the other can handle it via merge conflicts. Probably easier than ripping apart your changes ahead of time. Up to you though.

**Comment by BerginJay** [comment]
> You would need to update your PR to only include those changes. The other PR is waiting for merge conflict changes. I think if we get channel priority change in the Jenkinsfile and tests come black clean we can approve and merge this. We can favor whoever gets the PR merged first and the other can handle it via merge conflicts. Probably easier than ripping apart your changes ahead of time. Up to you though.

I'm not familiar with Jenkins, Docker and AWS. I added channel_priority to the Jenkinsfile and have been trying to test my own fork using Jenkins for the past few hours, but I keep getting this error: `ERROR: Could not find credentials matching ecr:us-west-2:Jenkins-Manager-Role`.

I used Pipeline script from SCM (i.e., Jenkinsfile in my own fork) with my AWS credential (just newly signed up). I think it may be that my account doesn't have permission for something like this, but I'm not sure exactly what the problem is. I googled the error but got no useful information. I think I might need help from someone familiar with this workflow or leave the testing to the maintainers.

Please find below the full test console output from Jenkins:
```
Started by user BerginJay
Obtained Jenkinsfile from git git@github.com:BerginJay/ISIS3.git
[Pipeline] Start of Pipeline
[Pipeline] node
Running on Jenkins in /Users/bojunjia/.jenkins/workspace/ISIS_channel_priority
[Pipeline] {
[Pipeline] stage
[Pipeline] { (Declarative: Checkout SCM)
[Pipeline] checkout
The recommended git tool is: git
using credential ecr:us-west-2:jbj0901
 > git rev-parse --resolve-git-dir /Users/bojunjia/.jenkins/workspace/ISIS_channel_priority/.git # timeout=10
Fetching changes from the remote Git repository
 > git config remote.origin.url git@github.com:BerginJay/ISIS3.git # timeout=10
Fetching upstream changes from git@github.com:BerginJay/ISIS3.git
 > git --version # timeout=10
 > git --version # 'git version 2.37.1 (Apple Git-137.1)'
using GIT_ASKPASS to set credentials Amazon ECR Registry:jbj0901-US_WEST_2
 > git fetch --tags --force --progress -- git@github.com:BerginJay/ISIS3.git +refs/heads/*:refs/remotes/origin/* # timeout=10
 > git rev-parse refs/remotes/origin/dev^{commit} # timeout=10
Checking out Revision dd203ad013cc68ca65856bcd3dfd7cbe991c390e (refs/remotes/origin/dev)
 > git config core.sparsecheckout # timeout=10
 > git checkout -f dd203ad013cc68ca65856bcd3dfd7cbe991c390e # timeout=10
Commit message: "Update Jenkinsfile setting channel_priority"
 > git rev-list --no-walk dd203ad013cc68ca65856bcd3dfd7cbe991c390e # timeout=10
[Pipeline] }
[Pipeline] // stage
[Pipeline] withEnv
[Pipeline] {
[Pipeline] withEnv
[Pipeline] {
[Pipeline] withDockerRegistry
[Pipeline] // withDockerRegistry
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // withEnv
[Pipeline] }
[Pipeline] // node
[Pipeline] End of Pipeline
ERROR: Could not find credentials matching ecr:us-west-2:Jenkins-Manager-Role
Finished: FAILURE
```

**Comment by Kelvinrr** [comment]
@BerginJay Your change looks good. Jenkins runs in the background for us and normally reports to the PR but I think the change to DGEC broke this. Looking at the logs it looks clean so I think this is good.

**Review by Kelvinrr — APPROVED** [review]

**Comment by BerginJay** [comment]
> @BerginJay Your change looks good. Jenkins runs in the background for us and normally reports to the PR but I think the change to DGEC broke this. Looking at the logs it looks clean so I think this is good.

Thanks, that's great. For the auto build check that was supposed to be available here, would resetting the webhook for this repo be helpful?

**Comment by BerginJay** [comment]
Just noticed the name order in [.zenodo.json](https://github.com/DOI-USGS/ISIS3/blob/dev/.zenodo.json). I am so sorry for breaking the alphabetical rule. I can open a new PR to fix it or leave it to you guys. I apologize.

**Comment by acpaquette** [comment]
@BerginJay Eventually we can update the hook but currently we are waiting on our jenkins service to be scanned, and if clean, exposed for public access. After that, we can get the build reports back

**Pull request merged**
